orderbook/src/orderbook.rs

```diff
@@ -1,10 +1,14 @@
+use contracts::GPv2Settlement;
+use futures::future::join_all;
+use futures::join;
```

Comment:
```suggestion
use futures::{future::join_all, join};
```
(might need cargo fmt after this)